builtin/plugins/coin/coin.go

```diff
@@ -417,6 +434,9 @@ func loadAccount(
 }
 
 func saveAccount(ctx contract.Context, acct *Account) error {
+	if ctx.FeatureEnabled(loomchain.CoinVersion1_2Feature, false) && (acct.Owner == nil || acct.Balance == nil) {
+		return ErrInvalidRequest
```

Comment:
This isn't a contract method, so returning `ErrInvalidRequest` doesn't make sense in this context, you should move the check to the caller of `saveAccount`